.gitignore

```diff
@@ -40,4 +40,5 @@
 
 # Ignore components vendored during tests
 tests/fixtures/scenarios/vendor/components/**
+tests/fixtures/scenarios/vendor-globs/components/**
 examples/demo-vendoring/components/**
```

Comment:
All these lines inhibit our ability to clean the component workspaces in between tests. We delete untracked files, but by adding them to gitignorr we cannot do that.